samples/sensor/fxls8974/sample.yaml

```diff
@@ -0,0 +1,25 @@
+common:
+  harness: console
+  harness_config:
+    type: multi_line
+    ordered: true
+    regex:
+      - ".*Booting.*"
+      - "^AX=\\W*[0]\\..*AY=\\W*[0]\\..*AZ=\\W*([9]|[1])([0-1]|\\.).*$"
+      - "^AX=\\W*[0]\\..*AY=\\W*[0]\\..*AZ=\\W*([9]|[1])([0-1]|\\.).*$"
+      - "^AX=\\W*[0]\\..*AY=\\W*[0]\\..*AZ=\\W*([9]|[1])([0-1]|\\.).*$"
+      - "^AX=\\W*[0]\\..*AY=\\W*[0]\\..*AZ=\\W*([9]|[1])([0-1]|\\.).*$"
```

Comment:
![image](https://github.com/user-attachments/assets/a462a50d-42df-41ca-9290-0a97f05ef658)
the regex match is wrong

Reply:
![obrazek](https://github.com/user-attachments/assets/06895dc9-6b27-4c35-af9b-0cbecff9c965)
Seems to work for me

Reply:
the regex match shall compatible to all the situations, stick to one situation is not reasonable.

Reply:
I've added a vector cross to the example, so the regex won't have to be stupid long, tested on 1064, test passes in all orientations.